model_analyzer/record/metrics_manager.py

```diff
@@ -167,26 +167,29 @@ def profile_server(self):
 
     def execute_run_config(self, run_config):
         """
-        Executes the run config. Returns obtained measurement. Also sends 
+        Executes the RunConfig. Returns obtained measurement. Also sends 
         measurement to the result manager
         """
 
+        model_run_config = run_config.model_run_configs()[0]
```

Comment:
Is this changing when we go to multi-model? And if so, add a comment here

Reply:
added comment